packages/time-utilities/src/lib/Timestamp.ts

```diff
@@ -0,0 +1,270 @@
+import { days, months, Time, tokens } from './constants';
+
+interface TokenResolver {
+	(time: Date): string;
+}
+
+const tokenResolvers = new Map<string, TokenResolver>([
+	// Dates
+	['Y', (time): string => String(time.getFullYear()).slice(2)],
```

Comment:
q: is the `: string` required per our lint rules?

Reply:
No.

Reply:
🧹